indico/modules/events/logs/client/js/components/EventLog.jsx

```diff
@@ -6,13 +6,45 @@
 // LICENSE file for more details.
 
 import React from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+
+import {Param, Translate} from 'indico/react/i18n';
 
 import LogEntryList from '../containers/LogEntryList';
 import Toolbar from './Toolbar';
+import {fetchLogEntries, setDataQuery, setPage} from '../actions';
+
+function DataQueryMessage() {
+  const dispatch = useDispatch();
+  const dataQuery = useSelector(state => state.logs.dataQuery);
+
+  function resetUrlFiltering() {
```

Comment:
`URL` please 🙈 

and in most places we use arrow functions when nesting functions inside other functions